include/lib/el3_runtime/cpu_data.h

```diff
@@ -24,12 +26,16 @@
 
 #endif /* AARCH32 */
 
+/* CPU_DATA_LOG2SIZE depends on cache line size (FIXME:clean this) */
+#ifndef CACHE_WRITEBACK_SHIFT
+#define CACHE_WRITEBACK_SHIFT		6
+#endif
 #if CRASH_REPORTING
-#define CPU_DATA_LOG2SIZE		7
```

Comment:
Hi @etienne-lms, the CACHE_WRITEBACK_SHIFT is not a platform macro whereas CACHE_WRITEBACK_GRANULE is a platform macro. Hence CPU_DATA_LOG2SIZE need to be derived from CACHE_WRITEBACK_GRANULE. But it is not easy to extract the trailing zeroes because this macro is required in assembly as well hence the __builtin_ctzl cannot be used here.

The other alternative is to remove this macro totally and use  `count_tz` macro in `_cpu_data_by_index()`  to extract the trainiling zeroes from CACHE_WRITEBACK_GRANULE.